### PR TITLE
change edit button location to match mockups

### PR DESCRIPTION
**Pull request by Rytiggy**

### CODE REVIEW

File: app/components/layer-wrapper/style.scss [hunk 1]
```diff
@@ -47,4 +47,6 @@
     }
 }
 
-
+.margin-20{
```
Comment (caneruguz) [inline app/components/layer-wrapper/style.scss]: Can you use the helper classes before doign this? we have m-md or mg-lt. They are not exactly 20: https://centerforopenscience.github.io/osf-style/style.html#layout-helper

File: app/components/layer-wrapper/template.hbs [hunk 2]
```diff
@@ -1,13 +1,13 @@
 <div id="layer{{index}}" class="layer-wrapper {{layer.component}} theme-{{theme.type}}" style="background-color:{{theme.background}}; color: {{theme.color}};" >
+    {{#if editMode}}
+        <button class="btn btn-outline-default-cus pull-right margin-20" style="color:{{theme.color}}; border-color:{{theme.color}};" {{action toggleProperty 'showSettings'}}>Edit section</button>
```
Comment (caneruguz) [inline app/components/layer-wrapper/template.hbs]: By the way what does 'cus' stand for?

Comment (caneruguz) [inline app/components/layer-wrapper/template.hbs]: Did you check that pull-right is behaving well with all layers?

Comment (Rytiggy) [inline app/components/layer-wrapper/template.hbs]: cus stands for custom, and pull-right is behaving as expected for all layers

**Pull request merged**